rosdep/base.yaml

```diff
@@ -226,6 +226,10 @@ beep:
   fedora: [beep]
   gentoo: [app-misc/beep]
   ubuntu: [beep]
+benchmark:
+  debian: [libbenchmark-dev]
+  fedora: [google-benchmark]
+  ubuntu: [libbenchmark-dev]
```

Comment:
The same goes for Ubuntu here, where this package is available in bionic and later, but not xenial (which is still supported and used by Kinetic).  A similar fix here would be:

```
ubuntu:
  xenial: null
  *: [libbenchmark-dev]
```

Reply:
Changed.